zone-service/src/test/java/org/cloudfoundry/identity/uaa/zone/ZoneServiceTests.java

```diff
@@ -57,6 +59,32 @@ public void testGetZoneDetails_NotFound() {
         assertTrue(exception.getMessage().contains("Zone not available."));
     }
 
+    @Test
+    public void testDeleteZone() {
+        IdentityZone identityZone = buildIdentityZone();
+        when(zoneProvisioning.retrieveByName(any())).thenReturn(identityZone);
+        ResponseEntity<?> response = zoneService.deleteZone(identityZone.getName());
```

Comment:
do we return ResponseEntity<?> from zoneService.deleteZone()? if yes then why I think we should return ResponseEntity<?> type response from controller only and for delete case we can have void return type in zoneService.deleteZone().